blueflood-core/src/main/java/com/rackspacecloud/blueflood/io/serializers/astyanax/HistogramSerializer.java

```diff
@@ -0,0 +1,42 @@
+/*
+ * Copyright 2013 Rackspace
+ *
+ *    Licensed under the Apache License, Version 2.0 (the "License");
+ *    you may not use this file except in compliance with the License.
+ *    You may obtain a copy of the License at
+ *
+ *        http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *    Unless required by applicable law or agreed to in writing, software
+ *    distributed under the License is distributed on an "AS IS" BASIS,
+ *    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *    See the License for the specific language governing permissions and
+ *    limitations under the License.
+ */
+
+package com.rackspacecloud.blueflood.io.serializers.astyanax;
+
+import com.netflix.astyanax.serializers.AbstractSerializer;
+import com.rackspacecloud.blueflood.io.serializers.metrics.HistogramSerDes;
+import com.rackspacecloud.blueflood.types.HistogramRollup;
+
+import java.nio.ByteBuffer;
+
+public class HistogramSerializer extends AbstractSerializer<HistogramRollup> {
+    private static final HistogramSerDes deSer = new HistogramSerDes();
```

Comment:
I'd recommend doing a private constructor on this.  Not sure if this kind of stuff was in the scope of the PR, but I'll comment on it.  The private constructor will prevent anyone from creating their own instance rather than using the singleton.